packages-ui/roosterjs-react/lib/ribbon/component/buttons/tableEditOperations.ts

```diff
@@ -0,0 +1,38 @@
+import RibbonButton from '../../type/RibbonButton';
+import { editTable } from 'roosterjs-editor-api';
+import { IEditor, TableOperation } from 'roosterjs-editor-types';
+import { TableEditOperationsStringKey } from '../../type/RibbonButtonStringKeys';
+
+type TableEditOperationsKey = 'deleteTable' | 'deleteRow' | 'deleteColumn';
+
+const tableEditOperationsLabel: Record<TableEditOperationsKey, string> = {
+    deleteTable: 'Delete Table',
+    deleteRow: 'Delete Row',
+    deleteColumn: 'Delete Column',
+};
+
+const tableEditOperations: Record<TableEditOperationsKey, TableOperation> = {
+    deleteTable: TableOperation.DeleteTable,
+    deleteRow: TableOperation.DeleteRow,
+    deleteColumn: TableOperation.DeleteColumn,
+};
+
+/**
+ * @internal
+ * "TableEditOperations" button on the format ribbon
+ */
+export const tableEdit: RibbonButton<TableEditOperationsStringKey> = {
```

Comment:
Do you want this become an office feature of roosterjs-react ribbon or just an entry of this feature for demo site?

If the latter, you can put this button into demo site path (similar with popout button) so that you can still use it in demo site but no need to publish it to other developers.